### ISSUE TITLE
Add file size information and downloaded percentage.

### ISSUE THREAD

**Issue by Weeeishy** [bbougot/Popcorn#140]
Something similar to this in the page of the movie/tv show.

https://i.gyazo.com/81e0a405b0079a6f07c942c4cdfe68b4.png

Also, an indicator of how much we downloaded so far with a percentage would be something nice to have

https://i.gyazo.com/9a56559a011387ad7b2e713f03f37d33.png

It would be a nice improvement with people who have a low download speed or limited bandwidth.

**Comment by bbougot** [comment]
Actually you can see the file size info on the I circle button in the top right of the movie page, it's a tooltip and you have to let the mouse hover it.

Regarding the downloading progress while playing, it's a nice to have, I will look at it.

**Comment by Weeeishy** [comment]
Thanks for clarifying. It actually looks much better than the one I linked above. Although, it only works for movies and not tv shows.

**Comment by bbougot** [comment]
Actually this tooltip is also available for each episode in shows.

**Comment by Weeeishy** [comment]
That's right, but it shows 0 for both seeds and peers and it doesn't show the size for every tv show I've tried so far. My bad for not clarifying that.

**Comment by bbougot** [comment]
Oh right, I've already seen this issue. I will dig into it as soon as possible. Thanks for the report 👍
